mappings.json

```diff
@@ -142,6 +142,18 @@
     "export": "capabilities",
     "deprecated": false
   },
+  {
+    "global": "Ember._Input",
+    "module": "@ember/component",
+    "export": "Input",
+    "deprecated": false
+  },
+  {
+    "global": "Ember._TextArea",
+    "module": "@ember/component",
+    "export": "TextArea",
```

Comment:
This export should actually be `Textarea`, we decided that it should match the capitalization of `<textarea>` which is one word (and it's the same as resolving it).

Reply:
updated!